post-type-archive-descriptions.php

```diff
@@ -241,11 +271,29 @@ function ptad_editor_field( $args ) {
 }
 
 /**
- * Output settings pages
+ * Output settings pages for standard CPT setup
  */
-function ptad_settings_page() {
+function ptad_settings_standard() {
 	$screen = get_current_screen();
 	$post_type = $screen->post_type;
+	
+	ptad_settings_page( $post_type );
+}
+
+/**
+ * Output settings pages for custom CPT setup
+ */
+function ptad_settings_custom() {
+	$page = $_GET['page'];
```

Comment:
This method of determining the post type should work for both top-level and nested admin pages, so I'm tempted to remove the check and then just use the `$_GET[]` var. Any objections?